src/main/java/org/jenkinsci/test/acceptance/po/Node.java

```diff
@@ -44,6 +44,8 @@ protected Node(Injector i, URL url) {
     public abstract String getName();
 
     public void setExecutors(int n) {
+        // On high Res screens the UI redraws, so best wait for numExecutors to be present
+        waitFor(by.path("/numExecutors"));
```

Comment:
I can't be sure about the rest of the pages, I have not seen any fails because of redrawing. But in the case of this test, yes. Redrawing were hiding the elements temporarily, I have seen waits (in the form of elasticSleep) on other tests to ensure JS generation is finished before interacting with the page, but personally I prefer to use `waitFor` instead of `elasticSleep` to not delay the tests more than needed. Also is more than probably that this kind of fail happends only on HD displays (I was unable to get any fail until my new laptop arrived) making it not very common and undetected by the CI server, but a problem is a problem and should be fixed.